application/console/migrations/m230515_162100_create_session_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use yii\db\Migration;
+
+class m230515_162100_create_session_table extends Migration
+{
+    // Use safeUp/safeDown to run migration code within a transaction
+    public function safeUp()
+    {
+        /*
+         * UserTable
+         */
+        $this->createTable(
+            '{{session}}',
+            [
+                'id' => 'char(40) primary key not null',
+                'expire' => 'int(11) null',
+                'data' => 'binary not null',
+            ],
+            "ENGINE=InnoDB DEFAULT CHARSET=utf8"
+        );
```

Comment:
```suggestion
        );
        $this->createIndex('idx_expire', 'session', 'expire', false);
```
The docs recommend adding an index in production. We may not scale large enough to notice the difference, but I suppose it can't hurt.